app/settings.py

```diff
@@ -99,3 +99,5 @@ def get_key(_key_name):
 
 # JWT configurations
 EQ_JWT_LEEWAY_IN_SECONDS = 120
+
+EQ_ROLE_PERMISSIONS = {'admin': ['flush']}
```

Comment:
I realise I suggested having roles and permissions, but I've thought about it some more and agree with the comments made. I think for now we should simplify this and go with a `flusher` role; we can expand on this with permissions should we need them at a later date.